.ko.yaml

```diff
@@ -0,0 +1 @@
+defaultBaseImage: gcr.io/distroless/static:latest
```

Comment:
Can we try `gcr.io/distroless/static:nonroot` just to see if it would work too ? :upside_down_face:

Reply:
done